lib/driver.js

```diff
@@ -54,6 +54,14 @@ module.exports = R.curry((server_url, Client) => {
       Client.put( DEFAULT_HEADERS, BucketUrl(bucket, docid), doc )
     )
 
+    // revid is the latest revision id needed to update an existing document
+  , upsert: R.curry((bucket, docid, revid, doc) => {
```

Comment:
yes, there will be an update function on the `got-couch` side.  but we should not be using that.  we should be using the methods from the `kuss` abstraction layer.